integration_tests/run_pyspark_from_build.sh

```diff
@@ -15,6 +15,7 @@
 set -ex
 
 SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+export SPARK_RAPIDS_DEFAULT_CONFIGS_PATH=${SCRIPTPATH}/target/spark-rapids-default-configs.json
```

Comment:
I think we can just have `spark-session.py` construct this path without this env variable

Reply:
I did this because I was not sure how to get the right path in `spark-session.py` even when the python script is executed in a different directory than the spark-rapids root. Or in some very random directory. Any idea how to do it in that python file?

Reply:
We can follow the pattern with other paths, add a CLI option https://github.com/NVIDIA/spark-rapids/blob/bdc563ee3b380ed25fe6a817a6a04085ea0005c9/integration_tests/conftest.py#L15

Reply:
Hmm with that approach, should the user provide the correct path? I'm not sure how it is better than the current approach. The current one automatically locates the JSON file regardless of the CWD.

Reply:
Oh did you mean that we could pass the right path in the pom file if the pytest accepts the JSON file path as its argument? Yeah that would work.

Reply:
the default will be ${SCRIPTPATH}/target/spark-rapids-default-configs.json 

but if the user needs to override it they can use a CLI param.

Reply:
Yeah, your suggestion makes sense now. I added a new option as you suggested.

From the scoping perspective, I appreciate your input, but this seems a bit out of the scope of the original PR. Your suggestion is to allow more flexible default configuration for integration test, which I agree could be useful, but it is not directly relevant to the original mismatching default configuration issue. I'd like to keep the scope as minimum as possible. We can always add such a feature later when we actually use it.